eng/Versions.props

```diff
@@ -8,8 +8,11 @@
     <VersionMajor>8</VersionMajor>
     <VersionMinor>0</VersionMinor>
     <VersionSDKMinor>2</VersionSDKMinor>
+    <!-- Use the feature version for each monthly servicng release -->
+    <!-- Use the patch version for intra-monthly releases and hotfixes -->
     <VersionFeature>00</VersionFeature>
-    <VersionPrefix>$(VersionMajor).$(VersionMinor).$(VersionSDKMinor)$(VersionFeature)</VersionPrefix>
+    <VersionPatch>00</VersionPatch>
```

Comment:
If I remember correctly, VersionPatch should always be a part of the VersionPrefix, but it should be removed if it's 00 when constructing the real version, right?

Reply:
It has to always be included. Otherwise 8.0.203, 8.0.20301 and 8.0.204 would not be in the right order of nuget precedence.

It's just that from the customer's perspective, we're going to treat "8.0.204" (what the customer types) as 8.0.20400 (what our package has)